web/styles/left_sidebar.css

```diff
@@ -603,6 +603,20 @@ li.active-sub-filter {
     border: 0.5px solid var(--color-border-unread-counter);
 }
 
+/* Don't show unread counts on views... */
+.top_left_inbox,
+.top_left_recent_view,
+.top_left_all_messages {
+    .unread_count {
+        visibility: hidden;
+    }
+    &.selected-home-view .unread_count {
+        visibility: visible;
+    }
+}
+
+/* ...unless it's the selected home view. */
```

Comment:
This comment should presumably be just above the `&.selected-home-view .unread_count` line.